doc/classes/Control.xml

```diff
@@ -972,7 +972,7 @@
 		<member name="clip_contents" type="bool" setter="set_clip_contents" getter="is_clipping_contents" default="false">
 			Enables whether rendering of [CanvasItem] based children should be clipped to this control's rectangle. If [code]true[/code], parts of a child which would be visibly outside of this control's rectangle will not be rendered and won't receive input.
 		</member>
-		<member name="custom_minimum_size" type="Vector2" setter="set_custom_minimum_size" getter="get_custom_minimum_size" default="Vector2(0, 0)">
+		<member name="custom_minimum_size" type="Vector2i" setter="set_custom_minimum_size" getter="get_custom_minimum_size" default="Vector2i(0, 0)">
```

Comment:
The title of your PR says you made Vector2i values be just Vector2, but here you are changing Vector2 to Vector2i. What was the reason for this change?

Reply:
This was discussed with Yuri in the dev chat, but in a nutshell: It's good for those functions to always take those values as integers.

It changes nothing for the main objective of this PR.